aws-js-containers/README.md

```diff
@@ -1,25 +1,25 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Easy container example
+# ECS Fargate Containers
 
 Companion to the tutorial [Provision containers on AWS](https://www.pulumi.com/docs/tutorials/aws/ecs-fargate/).
 
 ## Prerequisites
 
 To run this example, make sure [Docker](https://docs.docker.com/engine/installation/) is installed and running.
 
-## Running the App
+## Deploy the App
 
 Note: some values in this example will be different from run to run.  These values are indicated
 with `***`.
 
-1.  Create a new stack:
+Step 1:  Create a new stack
```

Comment:
Shouldn't "Step X" be a title? ([example](https://www.digitalocean.com/community/tutorials/how-to-create-a-cluster-of-docker-containers-with-docker-swarm-and-digitalocean-on-ubuntu-16-04#step-1-%E2%80%94-provisioning-the-cluster-nodes))